irohad/ametsuchi/impl/postgres_setting_query.hpp

```diff
@@ -24,21 +24,24 @@ namespace iroha {
                            logger::LoggerPtr log);
 
       expected::Result<
-          std::shared_ptr<const shared_model::validation::Settings>,
+          std::unique_ptr<const shared_model::validation::Settings>,
           std::string>
       get() override;
 
      private:
       expected::Result<
-          std::shared_ptr<const shared_model::validation::Settings>,
+          std::unique_ptr<const shared_model::validation::Settings>,
           std::string>
-      update(std::shared_ptr<shared_model::validation::Settings> base);
+      update(std::unique_ptr<shared_model::validation::Settings> base);
 
       std::unique_ptr<soci::session> psql_;
       soci::session &sql_;
 
       logger::LoggerPtr log_;
     };
+
+    const shared_model::interface::types::SettingKeyType
+        kMaxDescriptionSizeKey = "MaxDescriptionSize";
```

Comment:
such definition may lead to "multiple definition" error. it is better to either make it static or move to cpp file if possible (leaving an `extern` declaration here).

Reply:
Fixed